### PR TITLE
refactor `(): void =>` to `() =>`

### PR DESCRIPTION
**Pull request by MaxGraey**

### CODE REVIEW

**Comment by jtenner** [comment]
Thank you for this pull request! I will pull it before 2.7.0 release.

File: packages/assembly/assembly/internal/RTrace.ts [hunk 1]
```diff
@@ -79,6 +79,7 @@ export class RTrace {
   /**
    * This method returns the current number of active references on the heap.
    */
+  @inline
```
Comment (jtenner) [inline packages/assembly/assembly/internal/RTrace.ts]: These optimizations make the binary faster, however, it makes compile times slower.

By a lot. 

The aspect cli bottlenecks are all related to compile times and we avoid these sorts of micro-optimizatiosnss.

Comment (MaxGraey) [reply packages/assembly/assembly/internal/RTrace.ts]: reverted

**Comment by jtenner** [comment]
Closed by #168

**Pull request closed without merging**